<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConsulta</class>
 <widget class="QDialog" name="frmConsulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta de Licencias</string>
  </property>
  <widget class="QPushButton" name="btnConsultar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Consultar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblLicencias">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTipo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Licencia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de DÃ­as</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotal">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
